Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -578,12 +572,34 @@ void MemoryWidget::FindValue(bool next)
   if (next)
   {
     end = &ram_ptr[ram_size - search_for.size() + 1];
-    ptr = std::search(&ram_ptr[addr], end, search_for.begin(), search_for.end());
+
+    if (m_ignore_case->isChecked())
+    {
+      ptr = std::search(&ram_ptr[addr], end, search_for.begin(), search_for.end(),
+                        [](char ch1, char ch2) {
```

Comment:
Thoughts on storing this comparison in a local lambda and using it in both branches, rather than typing it out twice?

Reply:
Yeah that would look nicer.